src/coreclr/vm/jitinterface.cpp

```diff
@@ -11788,6 +11788,42 @@ void* CEEJitInfo::getFieldAddress(CORINFO_FIELD_HANDLE fieldHnd,
     return result;
 }
 
+void* CEEJitInfo::getFrozenHandleFromInitedStaticField(CORINFO_FIELD_HANDLE fieldHnd)
+{
+    CONTRACTL {
+        THROWS;
+        GC_TRIGGERS;
+        MODE_PREEMPTIVE;
+    } CONTRACTL_END;
+
+    void* ptr = nullptr;
+
+    JIT_TO_EE_TRANSITION();
+
+    bool isSpeculative;
+    if (getStaticFieldCurrentClass(fieldHnd, &isSpeculative) != NULL && !isSpeculative)
```

Comment:
The JIT/EE interface methods should not be called from within other JIT/EE interface methods like this (inside `JIT_TO_EE_TRANSITION`). Nesting `JIT_TO_EE_TRANSITION` does not work well with exception handling.

Reply:
I would just duplicate the part that this method needs.

Reply:
Thanks! Done